### PR TITLE
Benchmarks: Micro benchmark - Add graph mode in NCCL/RCCL benchmarks for latency metrics

### PR DESCRIPTION
**Pull request by yzygitzh**
**Description**
Revise NCCL/RCCL benchmarks to add latency metrics.

### CODE REVIEW

**Review by yukirora — APPROVED** [review]
seems this title does not match the content?

File: superbench/config/azure_ndmv4.yaml [hunk 1]
```diff
@@ -73,6 +73,17 @@ superbench:
             NCCL_IB_DISABLE: '0'
       parameters:
         ngpus: 8
+    nccl-lat:default:
```
Comment (cp5555) [inline superbench/config/azure_ndmv4.yaml:76]: Why do we change the default?

File: superbench/benchmarks/micro_benchmarks/cuda_nccl_bw_performance.py [hunk 2]
```diff
@@ -88,6 +88,12 @@ def add_parser_arguments(self):
             default=5,
             help='Number of warmup iterations. Default: 5.',
         )
+        self._parser.add_argument(
+            '--graph_iters',
+            type=int,
+            default=0,
+            help='Number of graph launch iterations. Set to 0 to disable graph mode. Default: 0.',
```
Comment (cp5555) [inline superbench/benchmarks/micro_benchmarks/cuda_nccl_bw_performance.py:95]: Also it's not related to latency as well

**Review by cp5555 — APPROVED** [review]

File: superbench/benchmarks/micro_benchmarks/cuda_nccl_bw_performance.py (hunk 2 above)
Comment (yzygitzh) [reply superbench/benchmarks/micro_benchmarks/cuda_nccl_bw_performance.py:95]: Fixed in PR title

**Pull request merged**